MetaMorpheus/EngineLayer/GlobalVariables.cs

```diff
@@ -451,42 +451,40 @@ private static void LoadRnaModifications()
 
         private static void LoadGlycans()
         {
-            OGlycanLocations = new List<string>();
-            NGlycanLocations = new List<string>();
+            OGlycanDatabasePaths = new List<string>();
+            NGlycanDatabasePaths = new List<string>();
 
             foreach (var glycanFile in Directory.GetFiles(Path.Combine(DataDir, @"Glycan_Mods", @"OGlycan")))
```

Comment:
it wasn't here before. but do we need to check that the directory and file exits? and throw an error if it is not there?